game-headed/src/main/resources/org/triplea/game/client/ui/javafx/fxml/AboutInformation.fxml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.text.Text?>
+
+<VBox fx:id="aboutSection" alignment="TOP_CENTER" xmlns="http://javafx.com/javafx/8.0.171" xmlns:fx="http://javafx.com/fxml/1" fx:controller="org.triplea.game.client.ui.javafx.screens.AboutInformation">
+   <children>
```

Comment:
Very minor, resolve as you see fit: I think it may make sense to leave out the `<children>` tag. `<children>` is a [default tag](https://docs.oracle.com/javase/8/javafx/api/javafx/fxml/doc-files/introduction_to_fxml.html#default_properties) for [Pane](https://openjfx.io/javadoc/11/javafx.graphics/javafx/scene/layout/Pane.html), which means that the tag does not have to be added when defining the sub-element(s). The same goes for `ImageView` reg. `image`. That said, I do not know how removing it may interact with scene builders like the Gluon scene builder.

Reply:
Yeah SceneBuilder adds those automatically, I don't think it'll  be worth removing